SPHINXsys/src/shared/sphinxsys_system/sph_system.h

```diff
@@ -66,21 +66,24 @@ namespace SPH
 				  size_t number_of_threads = std::thread::hardware_concurrency());
 		virtual ~SPHSystem(){};
 
+		void setRunParticleRelaxation(bool run_particle_relaxation) { run_particle_relaxation_ = run_particle_relaxation; };
+		bool RunParticleRelaxation() { return run_particle_relaxation_; };
+		void setReloadParticles(bool reload_particles) { reload_particles_ = reload_particles; };
+		bool ReloadParticles() { return reload_particles_; };
+		void setRestartStep(size_t restart_step) { restart_step_ = restart_step; };
+		size_t RestartStep() { return restart_step_; };
```

Comment:
My idea (rule) for naming member function is (simple or all ?) function with return value using Camel case, but void function with verb begin and then Camel case to indicate getting some value or doing some action.

Reply:
There is confusion between camelCase and PascalCase. Your suggestion is not really common and more confusing to apply especially here since your flags start with an action verb. The more usual alternative for getter/setters are to prepend get and set to the value name.